.gitignore

```diff
@@ -32,3 +32,5 @@ composer.lock
 nbproject
 Thumbs.db
 var/
+.cache/
+Build/.cache/
```

Comment:
When I am not mistaken, `.cache/` from above also inlcudes `Build/.cache/, so the last one can be omitted.